spectractor/extractor/targets.py

```
@@ -24,6 +26,24 @@
     _USE_NEW_SIMBAD = True
 
 
+def _get_cache_dir():
+    cache = os.path.join(os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))),
+                         "tests", "data", "cache", "astropy", "astroquery", "Simbad")
+    os.makedirs(cache, exist_ok=True)
+    return cache
+
+
+def _get_cache_file(tag):
+    filename = tag.replace("*", "").replace(" ", "_").replace(".", "_")
+    return filename
+
+
+def _clean_cache_dir():
+    cache = os.path.join(os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))),
+                         "tests", "data", "cache", "astropy", "astroquery", "Simbad")
+    os.rmdir(cache)
```

Does this work if the directory has content? We usually use `shutil.rmtree` for this because it can be told to ignore errors. Wouldn't removing this directory also remove the HD_111980.ecsv file?

---

OK I will use `rmtree`